backend/src/main/java/com/bakdata/conquery/apiv1/frontend/FEFilterType.java

```diff
@@ -11,5 +11,6 @@
 	REAL_RANGE,
 	NONE,
 	STRING,
-	MONEY_RANGE
+	MONEY_RANGE,
+	POSTAL_CODE
```

Comment:
Mir wäre es hier lieber mehrere Typen angeben zu können aber nachvollziehbar warum du das so gemacht hast.

Reply:
inwiefern wären mehrere Typen sinnvoller?

Reply:
Wie @thoniTUB und ich schon erklärt haben geht es einfach nicht so einfach, weil du hier ein Enum hast und enums nicht erweitert werden können über 2 Projekte hinweg. Und dieser code soll am Ende nach Eva.

Aber ich kann mir vorstellen, dass wir das enum abschaffen können und das ganze mit CPS-Types oder ähnliches hinbekommen. @thoniTUB was denkst du? Grundsätzlich könnte man ja auch an der stelle in configureFrontend direkt die klasse angeben und aus der wird dann über die CPSType-Annotation der Typ geholt. Vielleicht ist das sogar zuverlässiger?

Reply:
wird von @thoniTUB separat gelöst

Reply:
@Njimefo ich habe einen Vorschlag gemacht, welchen weg man gehen könnte und was dazu noch fehlt (dem Frontend mitteilen, wie es dargestellt werden soll und wie im Frontend die Eingabe validiert wird)

@awildturtok Vorschlag geht mehr in deine Richtung. Und ist erstmal einfacher umzusetzen und wahrscheinlich leichter mit @Kadrian abzusprechen weil es nur eine neue spezielle Implementierung eines Filters ist, im Gegensatz zu dem generischen Ansatz den ich vorgeschlagen hatte.

Aber ja das Enum musst du wegbekommen und da ist Fabians Vorschlag der praktikabelste.

Reply:
https://github.com/bakdata/conquery/pull/2246